module-domain/src/main/kotlin/site/yourevents/invitationthumnail/port/out/InvitationThumbnailPersistencePort.kt

```diff
@@ -0,0 +1,7 @@
+package site.yourevents.invitationthumnail.port.out
+
+import site.yourevents.invitationthumnail.domain.InvitationThumbnail
+
+interface InvitationThumbnailPersistencePort {
+    fun saveInvitationThumbnail(invitationThumbnail: InvitationThumbnail): InvitationThumbnail
```

Comment:
마찬가지로 JPARepository에서 기본으로 제공하는 메서드명으로 변경하는 것이 좋을 것 같아요!!
```suggestion
    fun save(invitationThumbnail: InvitationThumbnail): InvitationThumbnail
```

Reply:
위쪽에 남긴 답글과 동일합니다!! 생각 한번 부탁드립니다!!

Reply:
코멘트 확인해주시면 감사하겠습니다!! 👍

Reply:
마찬가지로 수정하였습니다!!